sdk/servicebus/service-bus/src/session/messageSession.ts

```diff
@@ -364,15 +375,15 @@ export class MessageSession extends LinkEntity {
         // SB allows a sessionId with empty string value :)
 
         if (this.sessionId == null && receivedSessionId == null) {
-          // Ideally this code path should never be reached as `createReceiver()` should fail instead 
+          // Ideally this code path should never be reached as `createReceiver()` should fail instead
           // TODO: https://github.com/Azure/azure-sdk-for-js/issues/9775 to figure out why this code path indeed gets hit.
           errorMessage = `No unlocked sessions were available`;
         } else if (this.sessionId != null && receivedSessionId !== this.sessionId) {
           // This code path is reached if the session is already locked by another receiver.
           // TODO: Check why the service would not throw an error or just timeout instead of giving a misleading successful receiver
-          errorMessage = `Failed to get a lock on the session ${this.sessionId};`
+          errorMessage = `Failed to get a lock on the session ${this.sessionId};`;
```

Comment:
I imagine the semi-colon at the end of the `errorMessage` string was meant to be after the string :smile:

Reply:
Fixed. I also had to double-check but it looks like that was in there before my PR :)